frontend/translations/en.po

```diff
@@ -4058,6 +4058,9 @@ msgstr "Group name"
 msgid "workspace.assets.libraries"
 msgstr "Libraries"
 
+msgid "workspace.assets.libraries.add"
```

Comment:
I think the key should be something `workspace.assets.add-library` .
Translation keys should have a maximum of 4 segments but preferible 3.
This is a minor issue, of course.

Reply:
Fixed